src/components/fields/checkbox-group/types.ts

```diff
@@ -12,14 +12,18 @@ interface IToggleOption extends IOption {
 	none?: boolean;
 }
 
+export type TCheckboxToggleLayoutType = "horizontal" | "vertical";
+
 type TCustomOptions =
 	| {
 			styleType: "default";
+			layoutType?: TCheckboxToggleLayoutType;
 	  }
 	| {
 			styleType: "toggle";
 			indicator?: boolean;
 			border?: boolean;
+			layoutType?: TCheckboxToggleLayoutType;
```

Comment:
trouble you to add `undefined` for all the optional properties

this is to support projects using `exactOptionalPropertyTypes` in tsconfig

```suggestion
			indicator?: boolean | undefined;
			border?: boolean | undefined;
			layoutType?: TCheckboxToggleLayoutType | undefined;
```